crates/sui-rpc-benchmark/src/bin/pull_grafana_logs.rs

```diff
@@ -220,8 +220,15 @@ async fn run() -> Result<(), Box<dyn Error>> {
         info!("Found {} logs for method: {}", count, method);
     }
 
-    // Write logs to file in ascending timestamp order
-    let file = File::create("sampled_read_requests.jsonl")?;
+    let file_path = env::var("FILE_PATH").unwrap_or_else(|_| ".".to_string());
```

Comment:
nit: `FILE_PATH` is quite a generic name for this environment variable -- what about `OUTPUT_DIR` or something?

It may also be worth considering using `clap` for argument parsing now -- you have quite a few arguments to this binary now.